romana/kubernetes/conductor.go

```diff
@@ -0,0 +1,76 @@
+// Copyright (c) 2016 Pani Networks
```

Comment:
@pritesh the problem is - `/romana` doesn't reflect that it's cli related directory and people will try to drop random stuff into it. So, in order to prevent it from becoming trash directory maybe we could move all cli stuff into `/pkg/cli` or `/pkg/util/cli`. ?